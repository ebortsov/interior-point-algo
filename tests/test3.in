3 1
1 2 0
1 1 1
8
2 2 4
0.0001
