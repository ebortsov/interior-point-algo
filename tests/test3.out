alpha = 0.50: [0.0000344 7.9999484 0.0000172]
alpha = 0.90: [0.0000041 7.9999908 0.0000052]
